### PR TITLE
Fixes #2760: Removed all unused resources from res folder

### LINKED ISSUE #2760: Remove unused resources
Use the Refactor action "remove unused resources" from Android studio to track all unused resources:

image

### PR DESCRIPTION
**Pull request by RajaVamsi11**
## Description

Removed all unused resources from res folder.
## Related issues and discussion
Fixes #2760

### CODE REVIEW

**Comment by RajaVamsi11** [comment]
@deniger Please review this PR.

**Comment by deniger** [comment]
some conflicts.
For this cleaning PR, we should merged as soon as it's done.
@teolemon we shall test it asap after conflict is managed

**Comment by RajaVamsi11** [comment]
@teolemon @deniger conflicts resolved. Can you please review this PR again?

**Comment by RajaVamsi11** [comment]
@deniger Please review this PR asap because as commits of translations increase the conflicts also increases as there are many strings files are changed.

File: app/src/main/res/values-ro/strings.xml [hunk 1]
```diff
@@ -827,12 +682,14 @@
     <string name="go_to_full_product_page">Go to product</string>
     <string name="no_additive_data_available_for_this_product">No additive data available for this product</string>
     <string name="no_nutrient_information_available_for_this_product">No nutrient information available for this product</string>
-    <string name="error_duplicate_listname">You already have a list with this name</string>
+    <string name="error_duplicate_<<<<<<< unused-resources
```
Comment (deniger) [inline app/src/main/res/values-ro/strings.xml]: @RajaVamsi11 is it a merge error ?

Comment (RajaVamsi11) [reply app/src/main/res/values-ro/strings.xml]: yes

Comment (RajaVamsi11) [reply app/src/main/res/values-ro/strings.xml]: Did not notice that. Thanks for committing.

**Pull request merged**